src/core/views/mobile-modal.js

```diff
@@ -60,12 +61,19 @@ class MobileModal extends Component {
     this.handleFormTheme = callIfEmbedIdMatches(this.handleFormTheme.bind(this), this.props.embedId)
     this.handleClose = this.handleClose.bind(this)
     this.setIframeRef = this.setIframeRef.bind(this)
+    this.wrapperRef = createRef()
   }
 
   componentDidMount() {
     const originalViewportContent = setMobileMetaViewport()
     this.setState({ originalViewportContent })
 
+    if (this.wrapperRef.current) {
+      this.setState({
+        height: `${document.documentElement.clientHeight - this.wrapperRef.current.offsetTop}px`,
```

Comment:
Same question here. You are using `offsetTop`, but the tab bar in screenshot is on the bottom. How does this work?

Reply:
The customer situation is that they override our css for the wrapper adding a top of 80px, that created a good edge case for testing this. The wrapper offsetTop is the distance of the wrapper from the top of the viewport, in this case would be 80 because it is forced by the customer, in our ideal case would always be 0 because our popup if fixed and has a top:0; So this applies only when there is a case similar to that one, in other situation will always return 0

Reply:
If customer did some modifications on their side I would suggest them to fix it themselves.

Reply:
I agree, as discussed since the customer is doing custom css, we suggested a simple css fix for their edge case https://jira.typeform.tf/browse/DIST-864 see comment